src/client/themes/cyberpunk/index.ts

```diff
@@ -58,18 +62,17 @@ const cyberpunkTheme: DefaultTheme = {
       },
     },
 
-    walletButton: {
-      background: '#0CA7C9',
-      color: '#FBD282',
-    },
+    // walletButton: {
+    //   background: '#0CA7C9',
+    //   color: '#FBD282',
+    // },
```

Comment:
Remove ?

Reply:
Removed in 37090db0d7efc2706a2f90189ac28479f41cded6